src/api.rs

```diff
@@ -1572,6 +1572,54 @@ impl Api {
         Ok(rv)
     }
 
+    /// List all issues associated with an organization and a project
+    pub fn list_organization_project_issues(
+        &self,
+        org: &str,
+        project: &str,
+        max_pages: usize,
+        query: String,
+    ) -> ApiResult<Vec<Issue>> {
+        let mut rv = vec![];
+        let mut cursor = "".to_string();
+        let mut requests_no = 0;
+
+        loop {
+            requests_no += 1;
+
+            let resp = self.get(&format!(
```

Comment:
Add query conditionally based on `if let Some(query) = query { ... }`

Reply:
Why don't we keep the `query: String` and leave the  `QueryArg` below?

Reply:
It's more about the intent rather than correctness. Whenever someone comes and read this code in the future, they should be able to see from the function definition itself that query is optional and the api will function without it as well.

Reply:
It makes really sense, thank you for the suggestion :) check it out